### PR TITLE
Rename top level module from automation to openwpm

### LINKED ISSUE #685: Rename package automation to openwpm
Since our package name is going to be OpenWPM our top level directory should have the same name.

### PR DESCRIPTION
**Pull request by ankushduacodes**
fixes #685

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ and the addon-sdk framework. We recommend against using this release as Firefox
 52ESR is no longer receiving security updates.
 
 Changes:
-  * The ``automation`` library can now be used with Python 3.4 or later,
```
Comment (vringar) [inline CHANGELOG.md:82]: Maybe don't change the old changelog.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ OpenWPM
 
 OpenWPM is a web privacy measurement framework which makes it easy to
 collect data for privacy studies on a scale of thousands to millions
-of websites. OpenWPM is built on top of Firefox, with automation provided
+of websites. OpenWPM is built on top of Firefox, with openwpm provided
```
Comment (vringar) [inline README.md:9]: This needs to stay the way it is.

File: docs/Platform-Architecture.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ The user-facing component of the OpenWPM platform is the Task Manager. The Task
 
 ## Instantiating a Task Manager
 
-All automation code is contained within the `automation` folder; the Task Manager code is contained in `automation/TaskManager.py`.
+All openwpm code is contained within the `openwpm` folder; the Task Manager code is contained in `openwpm/TaskManager.py`.
```
Comment (vringar) [inline docs/Platform-Architecture.md:9]: The first automation should remain

File: openwpm/TaskManager.py [hunk 4]
```diff
@@ -58,7 +58,7 @@ def load_default_params(
 class TaskManager:
     """User-facing Class for interfacing with OpenWPM
 
-    The TaskManager spawns several child processes to run the automation tasks.
+    The TaskManager spawns several child processes to run the openwpm tasks.
```
Comment (vringar) [inline openwpm/TaskManager.py:61]: Don't change this

**Review by vringar — CHANGES_REQUESTED** [review]

**Comment by ankushduacodes** [comment]
@vringar Its not letting me open these conflicts in web editor, and I think the new merged changes are causing this issue, In my opinion we may need to halt the merge to master branch before we merge this branch, and it will also case alot issues with old forked branches.
Best solution right it seems like to pause new merges and resume after this branch is merged, I think I will close this pull request and make a new one with newest master branch so there are no conflicts, What is your opinion on this?

**Comment by vringar** [comment]
You're right. Please go ahead and do as you suggested it.

**Pull request closed without merging**